### ISSUE TITLE
Port voxel-clientmc to run on nw.js?

### ISSUE THREAD

**Issue by deathcap** [voxel/voxel-clientmc#21]
http://nwjs.io (fka node-webkit) allows writing desktop apps using Node.js and WebKit, might be interesting to port voxel-clientmc to run on it. Of course, the main goal of voxel-clientmc is to run in the browser, but having a desktop app could be useful, if only for debugging/testing (e.g. directly connect using TCP/IP to servers, no [wsmc](https://github.com/deathcap/wsmc) websocket proxy needed)

**Comment by rom1504** [comment]
kind of related : https://github.com/Minecheesecraft/Cheese-Bot

**Comment by deathcap** [comment]
Interesting, if cheesebot was able to use mineflayer in nw.js, probably could just drop all the wsmc stuff for the nw port and use it natively, connecting to voxel.js. 

Or this possible alternative? https://github.com/atom/electron

**Comment by rom1504** [comment]
Yeah I hear electron is better. (no idea why, but people seem to prefer electron these days)
